jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/controllers/DslWeightedSwitchController.java

```diff
@@ -0,0 +1,107 @@
+package us.abstracta.jmeter.javadsl.core.controllers;
+
+import com.blazemeter.jmeter.control.WeightedSwitchController;
+import com.blazemeter.jmeter.control.WeightedSwitchControllerGui;
+import org.apache.jmeter.gui.util.PowerTableModel;
+import org.apache.jmeter.testelement.TestElement;
+import org.apache.jorphan.collections.HashTree;
+import us.abstracta.jmeter.javadsl.core.BuildTreeContext;
+import us.abstracta.jmeter.javadsl.core.testelements.BaseTestElement;
+import us.abstracta.jmeter.javadsl.core.threadgroups.BaseThreadGroup.ThreadGroupChild;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * DSL wrapper for {@link WeightedSwitchController}
```

Comment:
would be nice to explain further avoiding users to have to look into other sources to understand what is the purpose of this controller.

Reply:
Remember also adding @since in new classes and methods